docker-compose.jsonnet

```diff
@@ -224,6 +225,7 @@
     [if $.enabled.nginx then "nginx"]: {
       # mapping of services to internal ports for nginx to forward
       local forward_ports = {
+        thrimbletrimmer: 80,
```

Comment:
thrimbletrimmer isn't really a seperate service (it's just an aspect of the nginx service), so it doesn't really have a sensical "port" (and indeed this value here is unused). I see two options:

1. Set port here to `0` and have a comment like `// thrimbletrimmer is served directly by nginx and its port value is unused`

2. Don't put thrimbletrimmer in the `SERVICES` env var at all. Remove it from this list and instead pass in a seperate env var `THRIMBLETRIMMER` = `"true"` or `""`. Then in the nginx config generate script, add it after line 21 as eg. `[ -n "$THRIMBLETRIMMER" ] && echo -e "\t\tlocation = / { return 301 \$scheme://\$host/thrimbletrimmer/dashboard.html; }\n\t\tlocation /thrimbletrimmer { }"`

I think option 2 is a lot cleaner, both in terms of the code here and in terms of the code in generate-config.
Thrimbletrimmer is a component but it isn't a service, as it does not have its own process or container. It's better to make it its own thing than to try to shoehorn it into being like a service but then add special cases to the concept of a service.

Reply:
I did it this way just to get it working with minimal changes. I agree that it is a sufficiently different case from the other modules that it should be handled properly; but you know far better how to do it than I.